### PR TITLE
Mobile and desktop view

### PR DESCRIPTION
**Pull request by Juank628**
## Global Summit: Mobile and desktop view
In this pull request I did the following:
- Configure linters in github and local enviroment
- Created HTML and CSS for mobile and desktop view of Home page
- Created HTML and CSS for mobile and desktop view of About page
- Added Javascript for mobile menu
- Added Javascript for show and hide speakers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
```
Comment (Santiago220991) [inline README.md]: I think it is a good idea to remove this commented block of code. It makes your project looks messy. :+1: :smiley: :smiley:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+          <img
+            src="./img/summit_logo.png"
+            alt="summit logo"
+          />
+        </div>
+        <div class="footer-info">
+          <p class="footer-text">2022 Modern React Lima.</p>
+          <p class="footer-text">Some Rights Reserved.</p>
+        </div>
+      </footer>
```
Comment (Santiago220991) [inline index.html]: Kindly make sure that your project sticks as much as possible to the template provided. 

- The position of the text and the elements are different from the template on the desktop version. Kindly fix the position of the elements and add the missing text. :pray: :pray:  You could use lorem ipsum text for the missing text.

Your design
image

Figma design
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+            </p>
+          </section>
+        </div>
+
+        <button type="button" class="join-button">
+          Join React Global Summir 2022
+        </button>
+
+        <a class="program-link" href="#">See the whole program</a>
+      </section>
```
Comment (Santiago220991) [inline index.html]: Kindly make sure that your project sticks as much as possible to the Figma template provided. 

- In this section, the template has 5 five cards however in your project there are just 4. Please add one more card. :eyes: :eyes: 

Your design
image

Template design
image

- Also please make sure to add the hover effect shown in the template. There is a white border when you hover over every card. :pray: :pray: 

image

**Review by Santiago220991 — CHANGES_REQUESTED** [review]
Hi @Juank628, 
I'm Santiago your reviewer for this PR. :wave: :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
:heavy_check_mark: Good PR summary.
:heavy_check_mark: Followed GitHub flow.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Live version working.
:heavy_check_mark: Good presentation video.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
[OPTIONAL] I think it is a good idea to add some animations. It could help to make your project more dynamic. :+1: :smiley: :smiley: 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ha-manel — APPROVED** [review]
# STATUS: APPROVED ✅

## Hi @Juank628 👋

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉**



## Highlights:
* Well written PR description ✔️ 
* All tests passed ✔️ 
* Professional README file 💯 
* Your website looks awesome 🥇 Keep rocking 🚀 

## Optional suggestions:
- [ ] It's better if your commit messages had a unified structure. i.e, capitalized sentences in the imperative form, example: "Add linters" instead of "linters added"...
- [ ] Please consider adding the MIT file that you've mentioned in the README file because that link is broken 👍 


------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**